core/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -462,12 +462,9 @@ public boolean equals(Object other) {
      * <p>The LoadBalancer is responsible for closing unused Subchannels, and closing all
      * Subchannels within {@link #shutdown}.
      *
-     * <p>The default implementation calls {@link #createSubchannel(List, Attributes)}.
```

Comment:
Should we still specify that this method calls the other method?

Reply:
Sure. I have change the javadoc to refer to the other method.